### PR TITLE
[UIE-120] Replaced idContainers with useUniqueId.

### PR DESCRIPTION
**Pull request by evrii**
### What
- Replaced idContainers with useUniqueId.

### Why
- I am working on porting notifications to packages.

### Testing strategy
- [ ] 

 

[UIE-120]: https://broadworkbench.atlassian.net/browse/UIE-120?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/libs/notifications.js [hunk 1]
```diff
@@ -74,134 +73,130 @@ const NotificationDisplay = ({ id }) => {
...
-                    },
-                  },
-                  [icon('times', { size: 20 })]
-                ),
-              ]),
-              notifications.length > 1 &&
-                div(
-                  {
-                    style: { alignItems: 'center', borderTop: `1px solid ${baseColor()}`, display: 'flex', fontSize: 10, padding: '0.75rem 1rem' },
+  return div(
```
Comment (evrii) [inline src/libs/notifications.js:79]: This looks like a large change in Github, but I removed the two outer react-hyperscript-helper functions and replaced their usage of IdContainer with two uses of useUniqueId.

**Review by nawatts — APPROVED** [review]

File: src/libs/notifications.js (hunk 1 above)
Comment (nawatts) [reply src/libs/notifications.js:79]: Reviewing with whitespace changes ignored makes the diff much smaller.
https://github.com/DataBiosphere/terra-ui/pull/4609/files?w=1

Comment (evrii) [reply src/libs/notifications.js:79]: Oooo, I'll have to try to remember that one. Thank you.

**Review by msilva-broad — APPROVED** [review]
nice cleanup

**Pull request merged**